quodlibet/quodlibet/images/hicolor/Makefile

```diff
@@ -0,0 +1,82 @@
+QL_SOURCE=scalable/apps/quodlibet.svg
+QL_DEST=quodlibet.png
+
+QL_SYM_SOURCE=scalable/apps/quodlibet-symbolic.svg
```

Comment:
So here I should remove the targets and variables for the "-symbolic" icons? Meaning don't convert them to png? Why do you say can't be converted? Inkscape seems to have converted it to png. Do you mean they should not be converted?